### ISSUE TITLE
Text collor issues on non-reactive popup menu items

### ISSUE THREAD

**Issue by junaru** [vinceliuice/Qogir-theme#122]
Hi, thanks for the wonderful theme and its variants!

I believe there are some text color issues on non-reactive popup menu items. The text color matches the background or is transparent.

![example](https://user-images.githubusercontent.com/59161/81827359-5d5cd800-9541-11ea-8e30-ae31781b13d2.png)

I have made a sample extension to reproduce the issue:

````
$ git clone https://github.com/junaru/testext.git testext
$ cd testext
$ make install
````
Alt+F2, enter "r", press return (or restart gnome if using Wayland)

````
$ gnome-extensions enable testext@github.junaru.com
````

The issue is with elements that have "reactive: false" option: 
https://github.com/junaru/testext/blob/master/testext%40github.junaru.com/extension.js#L38

The text appears normal (grayed out) in both default and 3rd party gnome-shell themes.

**Comment by vinceliuice** [comment]
Thanks! I'll fix this

**Comment by junaru** [comment]
Looks like you fixed it in d6beb53778e2f3579e2f30aba804e95804c7b961, thanks!
